simulariumio/tests/converters/test_readdy_converter.py

```diff
@@ -13,20 +14,28 @@
         # 4 particles 3 frames
         (
             ReaddyData(
-                spatial_unit_factor_meters=1e-9,
                 box_size=np.array([20.0, 20.0, 20.0]),
                 timestep=0.1,
                 path_to_readdy_h5="simulariumio/tests/data/readdy/test.h5",
                 radii={"C": 3.0, "A": 2.0, "B": 2.0},
                 ignore_types=["E"],
                 type_grouping={"C": ["A", "D"]},
+                time_units=UnitData("ms", 1e-6),
+                spatial_units=UnitData("nm"),
             ),
             {
                 "trajectoryInfo": {
-                    "version": 1,
+                    "version": 2,
+                    "timeUnits": {
+                        "magnitude": 1.0,
+                        "name": "nanosecond",
```

Comment:
Is this supposed to match the above time_unit ("ms")?

Reply:
yes but 1e-6 ms is 1 ns, I'm testing to make sure the units get simplified correctly